app/models/concerns/smooch_messages.rb

```diff
@@ -291,16 +291,20 @@ def bundle_list_of_messages_to_items(list, last)
         if message['type'] == 'text'
           # Get an item for long text (message that match number of words condition)
           if message['payload'].nil?
-            contains_link = Twitter::TwitterText::Extractor.extract_urls(message['text'])
+            contains_link = self.extract_url(message['text'])
```

Comment:
Suggestion: `contains_link` is a variable name that suggests that its type is `boolean`. I think it returns a `Link` object? If so, I suggest to rename it to something like `link_from_message`, something like that.